### DISCUSSION TITLE
Is it possible to deploy Meilisearch on my own MacBook?

### DISCUSSION THREAD

**Discussion by ghost** [meilisearch/product#742] [Feedback & Feature Proposal]
I have always been very interested in Meilisearch. Its elegance and efficiency have deeply impressed me. Therefore, when I was looking for an open-source search solution for my local documents (mainly Markdown notes in Obsidian), it was the first one that came to my mind.

I am curious if this project can run smoothly on my MacBook Air (M2, 24GB of memory)? If so, what approximate amount of system resources would it require?

Best regards!

**Comment by irevoire** [comment]
Hey @yoghurtlee-thu, yes, no issue at all with that.
You're way above the minimum spec required to run Meilisearch.

> If so, what approximate amount of system resources would it require?

It entirely depends on what you're trying to index, but I would expect Meilisearch to use close to zero resources for notes you're taking by hand.

Let us know how it goes if you do it

**Reply by ghost** [reply]
Hello @irevoire , thank you for your kind reply.

I tried deploying Meilisearch on my MacBook, and it only used about 150MB of memory during runtime **with the JSON index I tested**. This is indeed an astonishingly lightweight product. However, since I am not familiar with databases and also do not know how to efficiently write a JSON index for Markdown files, I still have a way to go before I can use Meilisearch to manage my notes.

Thank you again for your outstanding work. I hope more people will apply this powerful tool to personal knowledge management and share their experiences :)

**Reply by papatistos** [reply]
Have you made any progress in this intriguing endeavour?
